modules/chronograf-server/outputs.tf

```diff
@@ -1,7 +1,23 @@
-output "public_ip" {
-  value = "${aws_instance.chronograf_server.public_ip}"
+output "asg_name" {
+  value = "${aws_autoscaling_group.autoscaling_group.name}"
+}
```

Comment:
Do any tests or docs need to be updated now that Chronograf is no longer outputting an IP address? That is, how do you access it?

Reply:
Noted. Will update the README in the upcoming PR for multi-cluster examples and test